src/Main.java

```diff
@@ -1,9 +1,34 @@
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
+        boolean play = true;
+        String ongeldig = "Ongeldige invoer.";
+        int[] numeric  = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+        String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul", };
 
-        // Hier mag je je code schrijven voor de hoofd-opdracht
+        Translator translator = new Translator(alphabetic, numeric);
+        Scanner scanner = new Scanner(System.in);
 
+        while (play) {
+            System.out.println("Type 'x' om te stoppen \nType 'v' om te vertalen");
+            String input = scanner.nextLine();
 
+            if (input.equals("x")) {
+                play = false;
+            } else if (input.equals("v")) {
+                System.out.println("Type 'x' om te stoppen \nType 'v' om te vertalen");
```

Comment:
Door deze regel wordt er opnieuw "Type 'x' om te stoppen, Type 'v' om te vertalen" geprint wanneer de input "v" is. Je krijgt daardoor een soort eindeloze loop met steeds dezelfde zin. Als je in deze regel "Type een cijfer in van 0 t/m 9" zet in plaats van nog een keer "Type 'x' om te stoppen \nType 'v' om te vertalen" doet hij het volgens mij helemaal goed! Ik kan me voorstellen dat dit gewoon een kopieerfoutje was 😄